modules/imgproc/src/min_enclosing_triangle.cpp

```diff
@@ -319,29 +318,12 @@ namespace minEnclosingTriangle {
 static void findMinEnclosingTriangle(cv::InputArray points,
                                      CV_OUT cv::OutputArray triangle, CV_OUT double &area) {
     std::vector<cv::Point2f> resultingTriangle, polygon;
-
-    createConvexHull(points, polygon);
+    CV_Assert(points.getMat().checkVector(2) > 0);
```

Comment:
Lets remove this check, because `convexHull` has similar checks inside through `total = points.checkVector(2)`

Or replace it with `CV_Assert(!points.empty());`

Reply:
replaced `CV_Assert(points.getMat().checkVector(2) > 0)` with `CV_Assert(!points.empty())`